src/pageObject.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
-
+  constructor(url, slug) {
+    super(url);
+    this.url = url + slug;
```

Comment:
The constructor in the ArticlePage class should modify the original `url` by adding `slug` at the end. Currently, it reassigns `this.url` to `url + slug`, which is correct. Ensure that this behavior aligns with the task requirements.